packages/webpack-utils/.npmrc

```diff
@@ -0,0 +1,2 @@
+registry=https://registry.npmjs.org/
```

Comment:
Isn't this the default? If so, do we need this?

Reply:
oh wow, I keep copying stuff that I don't really need!